apps/ledger-live-mobile/src/components/RootNavigator/RequestAccountNavigator.tsx

```diff
@@ -10,7 +10,7 @@ import AddAccountsNavigator from "./AddAccountsNavigator";
 import StepHeader from "../StepHeader";
 import type { RequestAccountNavigatorParamList } from "./types/RequestAccountNavigator";
 
-const totalSteps = "2";
+const totalSteps = "3"; // FIXME: no. of steps varies depending on the flow and whether e.g. you have an account.
```

Comment:
This flow is used across many places in the app, have we test them?
LL Stake flows: market, assets, account page, main stake button menu. Not sure, if it is the same flow that we are using in buy-sell too.
We do not want to break something that it is working fine.
@kallen-ledger I think was the last people who was working with this.